docs/pds4_create_xml_index.rst

```diff
@@ -18,34 +18,45 @@ and accessing structured data within PDS4-compliant datasets.
 XPath Syntax and Structure
 --------------------------
 
-Before using the tool, it is imperative that the user becomes comfortable with
-the XPath language and how it is parsed with ``lxml``.
-
-When elements are scraped from a label, they are returned in a format akin to a
-filepath. The absolute XPath contains all parent elements up to the root element
-of the label. Each element after a certain depth can also contain predicates:
-numbers surrounded by square brackets. These predicates give information on the
-location of the element in the label file, in relation to surrounding elements.
-
-However, this module returns XPath headers that have been reformatted from XPath
-syntax.
-
-- Namespaces are replaced by their prefixes. Namespaces are URIs that identify
-  which schema an element belongs to. For readability, the full URI's are
-  replaced by their prefixes.
-
-- Square brackets are replaced by angled brackets. This module utilizes ``glob``
-  syntax for finding label files and filtering out elements/attributes. Because
-  square brackets have meaning within ``glob`` statements, they needed to be
-  replaced with angled brackets.
-
-- The values within the predicates have been renumbered. In XPath syntax,
-  predicates are used to determine the location of an element in relation to its
-  parent. While this is useful in other applications, this logic fails if
-  multiples of the element and its parent appear within the document. Even if
-  the elements all have different values, all of their XPaths would be the same.
-  Instead, the predicates are renumbered to reflect which instance of the
-  element the value is represented by.
+This tool generates headers that represent the position of each element in the XML
+hierarchy in a manner similar to `the standard XPath format <https://developer.mozilla.org/en-US/docs/Web/XPath>`_. 
+While familiarity with XPath syntax is beneficial, it is not necessary to use this tool
+effectively. Note that for simplicity we call our headers XPaths throughout this document
+and within the command line arguments, despite some minor syntax differences. Below we
+describe the syntax we use for our headers. An XPath header is read from left to right,
+starting with the root element and moving through each subsequent child element. Each
+element in the path is separated by a forward slash (/). If an element has multiple
+instances within a single parent, predicates (numbers within angle brackets) are used to
+specify the exact instance, such as ../pds:Observation_Area<1>/pds:version_id<1>, which
```

Comment:
These paths should be in double-back-ticks. Also version_id and Observation_Area.